module/VuFind/src/VuFind/ILS/Logic/AvailabilityStatus.php

```diff
@@ -49,6 +49,13 @@ class AvailabilityStatus implements AvailabilityStatusInterface
      */
     protected int $availability;
 
+    /**
+     * Item visibility in holdings tab
+     *
+     * @var bool
+     */
+    protected bool $visibility = true;
```

Comment:
Does `$visibility` mean the same thing as `visibility in holdings tab`?  It seems like these fake items *are* visible in the search results, in the sense of having a presence there and saying to check availability on the item page.  There might be a more specific name, although I'm not sure `$visibilityInHoldingsTab` is better.

Reply:
Now that you mention it, @maccabeelevine, I think there's an important detail missing: this property can be set, but it's not actually used for anything. I think the isVisibleInHoldings() method needs adjustment too, and if that's the case, giving the property a more specific name definitely makes sense.

Reply:
Oh, nuts — that was sloppy of me. Yes, there is [currently code in AvailabilityStatus](https://github.com/vufind-org/vufind/blob/dev/module/VuFind/src/VuFind/ILS/Logic/AvailabilityStatus.php#L89-L98) that I forgot to change to make `setVisibility()` actually have an effect. Will fix.

If we decide to change the name of this method, I think the only place it is used is in [View/Helper/Root/Holdings::holdingIsVisible()](https://github.com/vufind-org/vufind/blame/dev/module/VuFind/src/VuFind/View/Helper/Root/Holdings.php#L75).

Lastly, I went with `$visibilityInHoldingsTab` as a more descriptive name.

Reply:
Looks much better to me, though I haven't re-tested yet (I'm holding off on doing another round of testing on my end until @maccabeelevine's testing is wrapped up and we've reached a conclusion about whether to make the behavior configurable... that way I'll keep my eyes fresher for the (hopefully) final round of testing). :-)

Reply:
Thanks, Demian...sounds like a good plan to me.